Utils/Dataflow/pyDKB/common/custom_readline.py

```diff
@@ -0,0 +1,13 @@
+def custom_readline(f, newline):
+    buf = ""
+    max_buf_size = 1
+    while True:
+        while newline in buf:
+            pos = buf.index(newline)
+            yield buf[:pos]
+            buf = buf[pos + len(newline):]
+        chunk = f.read(max_buf_size)
+        if not chunk:
+            yield buf
+            break
```

Comment:
Here it does return the last data read from `f` even if they are not terminated with EOM.
On the one hand, if it is a full message, we will be able to parse it (JSON or whatever); if not -- an error will occur and the message will be skipped.
On the other hand, if the data format does not allow to tell full message from its part (for text messages, for example), we will end up with wrong data.

Currently we use JSON, so it can be left as is, yet at least in the docstrings it should be noted -- that the last `line` can be incomplete, if the input data flow was interrupted in the middle of data writing.